pkg/packagekit/updates.jsx

```diff
@@ -553,6 +569,19 @@ class OsUpdates extends React.Component {
             case "applying":
                 return <ApplyUpdates transaction={this.state.applyTransaction}/>
 
+            case "updateSuccess":
+                return <AskRestart onRestart={this.handleRestart} onIgnore={this.loadUpdates} />
+
+            case "restart":
+                return (
+                    <div className="blank-slate-pf">
+                        <div class="blank-slate-pf-icon">
+                            <div className="spinner spinner-lg"></div>
+                        </div>
+                        <h1>{_("Restarting")}</h1>
+                        <p>{_("You will get disconnected soon. You can reconnect and resume your work after the server restarted.")}</p>
```

Comment:
Works for me. Garrett may have some opinion too.

wrt. the "Disconnected" screen: That is indeed not centered, at least not with the narrow screen width I used (in order to keep the screenshots small). That's an independent bug, I filed it as issue #7106 to not forget.